### PR TITLE
Add sample dashboard image

### PR DESCRIPTION
**Pull request by marksou**

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-pump/tyk-pump-configuration/datadog.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ url: /tyk-configuration-reference/tyk-pump-configuration/datadog/
 
 The Tyk Pump can be configured to send your API logs to [Datadog](https://www.datadoghq.com/) which then allows you to build [dashboards](https://docs.datadoghq.com/integrations/tyk/#dashboards) for your data.
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-pump/tyk-pump-configuration/datadog.md]: Can you please change it to "which then allows you to build [dashboards](https://docs.datadoghq.com/integrations/tyk/#dashboards) **with various metrics** based on your traffic in Tyk."

**Review by letzya — APPROVED** [review]

**Comment by letzya** [comment]
@marksou for this one maybe we should merge to LTS and 3.1 as well?

**Comment by marksou** [comment]
@letzya it has been.

**Comment by letzya** [comment]
@marksou are you sure? https://tyk.io/docs/3.1/tyk-configuration-reference/tyk-pump-configuration/datadog/

**Comment by marksou** [comment]
@letzya according to GitHub it has been. Will have another look in the morning.

**Pull request merged**